grails-app/services/org/pih/warehouse/data/LoadDataService.groovy

```diff
@@ -31,21 +32,21 @@ import java.text.SimpleDateFormat
 @Transactional
 class LoadDataService {
 
-    def locationDataService
+    def locationImportDataService
+    def productSupplierImportDataService
```

Comment:
I must have missed this one.
In general, I agree with the good practices of defining types for variables, but in cases of defining services I don't think it makes that much difference since we are not passing it as a variable or doing anything with it where the type would be important. 
And for intelisense, I believe the IDE handles in perfectly find.
But I agree, when possible we should define types for eveyting.